doc/i2c_bus_lock_prevention/i2c_bus_lock_prevention_hld.md

```diff
@@ -0,0 +1,425 @@
+|5|NOTICE|Reset all i2c mux devices to recover i2c bus lock.| |None|
+|6|NOTICE|Set all of the fans to full speed.| |None|
+|7|NOTICE|Stop all i2c daemons.| |None|
+|8|NOTICE|Start scanning for i2c faulty devices.| |None|
+|9|NOTICE|Finish scanning for i2c faulty devices.The isolation list is updated.| |None|
+|10|NOTICE|Resume all i2c daemons.| |None|
+|11|ERROR|Fatal error! Fail to stop all i2c daemons.| |Run 'show techsupport' and provide the dump file to the community with the reported issue.|
+|12|ERROR|Fatal error! Fail to reset all i2c mux devices.| |Run 'show techsupport' and provide the dump file to the community with the reported issue.|
+|13|ERROR|Fatal error! Fail to set all of the fans to full speed.| |Run 'show techsupport' and provide the dump file to the community with the reported issue.|
+|14|WARNING|Init representative devices list error.| |Run 'show techsupport' and provide the dump file to the community with the reported issue.|
+|15|INFO|Detect i2c bus health with I2C cmd '{}'.|Detect i2c bus health with I2C cmd 'sudo i2cget -f -y 0 0x77 0x0|None|
+|16|DEBUG|{}: i2c get successfully on attempt {}/{}.|EEPROM: i2c get successfully on attempt 1/3.|None|
+|17|NOTICE|{}: i2c get failed on attempt {}/{}.|EEPROM: i2c get failed on attempt 1/3.|None|
+|18|WARNING|{}: an internal error occurs on attempt {}/{}.|EEPROM: an internal error occurs on attempt 1/3.|Run 'show techsupport' and provide the dump file to the community with the reported issue.|
+|19|NOTICE|Execute cmd to stop i2c daemon: '{}'.|Execute cmd to stop i2c daemon: 'sudo systemctl stop as9716-32d-platform-monitor-fan.service'.|None|
+|20|ERROR|Failed to execute cmd to stop i2c daemon: '{}'.|Failed to execute cmd to stop i2c daemon: 'sudo systemctl stop as9716-32d-platform-monitor-fan.service'.|Run 'show techsupport' and provide the dump file to the community with the reported issue.|
+|21|NOTICE|Execute cmd to start i2c daemon: '{}'.|Execute cmd to start i2c daemon: 'sudo systemctl start as9716-32d-platform-monitor-fan.service'.|None|
+|22|ERROR|Failed to execute cmd to start i2c daemon: '{}'.|Failed to execute cmd to start i2c daemon: 'sudo systemctl start as9716-32d-platform-monitor-fan.service'.|Run 'show techsupport' and provide the dump file to the community with the reported issue.|
+|23|ERROR|Failed to add {}({}:{}) into the isolation list.|Failed to add Ethernet1(77-2-72-1:25-0050) into the isolation list.|1. Replace the transceiver inserted in the port mentioned by the message. 2. Run 'show techsupport' and provide the dump file to the community with the reported issue.|
+|24|ERROR|Failed to remove {}({}:{}) into the isolation list.|Failed to remove Ethernet1(77-2-72-1:25-0050) into the isolation list.|1-a. First option: remove the transceiver inserted in the port mentioned in the message. 1-b. Second option: reboot the device. 2. Run 'show techsupport' and provide the dump file to the community with the reported issue."|
+
+
+#### 6.5. Platform Driver

```

Comment:
Could you please elaborate the platform driver part?. What if the platform uses kernel driver like i2c-mux-pca954x?.